python/foglamp/services/core/syslog_parser.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+from pyparsing import Word, alphas, Combine, nums, string, Regex
+from time import strftime
+
+__author__ = "Praveen Garg"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_help = """
+    -------------------------------------------------------------------------------
+    | GET             | /foglamp/syslog                                             |
+    -------------------------------------------------------------------------------
```

Comment:
better to expose as REST api end point with filter  ?source=storage (case insensitive)

Reply:
please remove this help doc, its for api (endpoints) only